### ISSUE TITLE
Problems with default startup

### ISSUE THREAD

**Issue by githubmui** [crate/crate#6129]
CrateDB version: 2.1.5
OS: Windows

Problem 1: When starting crate.bat in "bin" folder, following error is shown: "ERROR: E is not a recognized option"
Problem 2: How to prevent following error: "Failed to locate the winutils binary in the hadoop binary path" on windows?
Problem 3: How to suppress warning: "SIGUSR2 signal not supported on Windows 7" ?

Thx for help!

**Comment by mikethebeer** [comment]
@githubmui Do you start CrateDB out of `cmd.exe` or `Powershell` ? In Powershell you can pass system properties by putting the properties into double quotes:

```sh
.\bin\crate "-Ccluster.name=crate -Cpath.data=D:\c1"
```

**Problem 2**: The binary distribution of Apache Hadoop 2.2.0 release does not contain some Windows native components (like winutils.exe, hadoop.dll etc). These are required (not optional) to run Hadoop on Windows. https://www.elastic.co/guide/en/elasticsearch/plugins/master/repository-hdfs-usage.html

**Problem 3**: The graceful stop signal is not supported in Windows. For development and testing its totally fine to run it on Windows. However, for running a CrateDB cluster in production we recommend to run it in a UNIX environment.

**Comment by githubmui** [comment]
@1: I started CrateDB directly out of cmd.exe. 
Starting "crate.bat" without any additional parameter leads to this error.

@2: If no Apache Hadoop is needed, how to get rid of this error? 
Is there e.g. some configuration to disable Apache Hadoop?

**Comment by rps-v** [comment]
Also failing to start Crate (`cmd.exe` or `Powershell`, doesn't matter - same error) :
```
C:\crate-2.1.5\bin> .\crate.bat
starts CrateDB

Option                Description
------                -----------
-C <KeyValuePair>     Configure a setting
-V, --version         Prints CrateDB version information and exits
-d, --daemonize       Starts CrateDB in the background
-h, --help            show help
-p, --pidfile <Path>  Creates a pid file in the specified path on start
-q, --quiet           Turns off standard ouput/error streams logging in console
-s, --silent          show minimal output
-v, --verbose         show verbose output
ERROR: E is not a recognized option
```
```
C:\crate-2.1.5\bin> .\crate
starts CrateDB

Option                Description
------                -----------
-C <KeyValuePair>     Configure a setting
-V, --version         Prints CrateDB version information and exits
-d, --daemonize       Starts CrateDB in the background
-h, --help            show help
-p, --pidfile <Path>  Creates a pid file in the specified path on start
-q, --quiet           Turns off standard ouput/error streams logging in console
-s, --silent          show minimal output
-v, --verbose         show verbose output
ERROR: E is not a recognized option
```

How to start Crate with old behavior (loading options from config) ?

**Comment by mikethebeer** [comment]
@rps-v @githubmui thanks for highlighting this. This issue has been introduced with a recent commit that came into 2.1. Sorry for this. A PR is already out https://github.com/crate/crate/pull/6142 so that this should be fixed in the next version of 2.1. For a quick fix you could adapt your `bin/crate.bat` like mentioned in the PR.
